python/paddle/fluid/tests/unittests/test_traced_layer_err_msg.py

```diff
@@ -174,5 +180,25 @@ def test_linear_net_with_none(self):
                                                                         [in_x])
 
 
+class TestTracedLayerSaveInferenceModel(unittest.TestCase):
+    """test save_inference_model will automaticlly create non-exist dir"""
+
+    def setUp(self):
+        self.save_path = "./nonexist_dir/fc"
+        import shutil
```

Comment:
NIT (Not Important): However, since it is not so important and this PR needs many approves. I will approve it and you can change in next PR.

Python officially suggests to put imports at top of file, I think you should follow it in your case. 
https://www.python.org/dev/peps/pep-0008/#imports

![image](https://user-images.githubusercontent.com/7913861/113804315-e5b71d80-9790-11eb-9469-1c8e3671972c.png)

Let me explain my understanding of the pros and cons of importing at top of a file or in a function.

Importing at top of a file:
1. More consistent with other programming language.
2. We group imports all together and easy to organize

There are several cases I think we can import in a function:
1. Conditional import, e.g.
```
if something:
    import xxx
```
2. You would like your function code to work correctly after copy the function code to other file (other file may not contain that import)

But I don't think here we meet the cases for importing in a function :-) so I would suggest to import at the top.